src/extendedstorage.cpp

```diff
@@ -706,15 +690,20 @@ Notebook::Ptr ExtendedStorage::createDefaultNotebook(QString name, QString color
     }
     if (color.isEmpty())
         color = "#63B33B";
-    uid = "11111111-2222-3333-4444-555555555555";
 #else
     if (name.isEmpty())
         name = "Default";
     if (color.isEmpty())
         color = "#0000FF";
 #endif
-    Notebook::Ptr nbDefault = Notebook::Ptr(new Notebook(uid, name, QString(), color, false, true, false, false, true));
-    addNotebook(nbDefault, false);
+    Notebook::Ptr nbDefault = Notebook::Ptr(new Notebook(uid.mid(1, uid.length() - 2), name, QString(), color, false, true, false, false, true));
```

Comment:
> Also existing thing, but wonder what's this omission of the first and last character on the uid about.

Looking at the documentation, it's because of braces. There is a formatting option not to get the braces in the `toString()` method, but it has been added in Qt5.11.